ci-requirements-py3.txt

```diff
@@ -0,0 +1,2 @@
+astroid==2.2.5
+pylint==2.3.1
```

Comment:
Can the version consider Python 2?

I wonder if pylint 2 claims errors which might break Python 2, but good for Python 3.

Reply:
> I wonder if pylint 2 claims errors which might break Python 2, but good for Python 3.

 I can't say it won't happen.
Your point is that we should have the same pylint version for python2/3, right?

Since exception log is no impact to the results, we can upgrade those version when we drop python2 support.

Reply:
Yes.
We support both of them by one codebase.

---

btw, we have https://github.com/appium/python-client/blob/master/development.txt .
It refers to `ci-requirements.txt`. we should update here, too.